crypto/fipsmodule/service_indicator/service_indicator_test.cc

```diff
@@ -71,6 +71,8 @@ static const uint8_t kAESCBCCiphertext[64] = {
 TEST(ServiceIndicatorTest, BasicTest) {
   int approved = 0;
   uint32_t serviceID = 0;
+  uint64_t counter = awslc_fips_service_indicator_get_counter();
+  ASSERT_EQ(sizeof(counter), (uint64_t)8);
```

Comment:
how did you come up with 8? Shouldn't the counter be 0 since we haven't called any service yet?

Reply:
The test here is only asserting the field size of the counter, which is 8 bytes. I removed the test for the `counter = 0` because any previous tests running on the test thread still increment the service indicator counter. We also changed the logic of the counter to `!=`, so the initial value shouldn't really matter anymore.